### DISCUSSION TITLE
Problema con Retrofit

### DISCUSSION THREAD

**Discussion by jfloresmaquera** [dds-jv/foro#258] [TPA - 2021] [answered]
Buenas noches, con mi grupo estuvimos intentando realizar la integracion de la API en nuestro TPA mediante la herramienta de Retrofit. Agregamos las dependencias dentro del POM, usamos de guia el repositorio que se uso en el seminario, pero aun asi a la hora de importar la clase Retrofit no nos lo toma.
Si nos pudieran dar una mano con esto seria estupendo.

**Comment by nanderson87** [comment, marked as answer]
@jfloresmaquera 

Nos podrás pasar una captura de pantalla del pom.xml con el agregado de la dependencia de `Retrofit` y el resultado de haber ejecutado el comando  `mvn dependency:resolve` dentro del proyecto?

**Reply by jfloresmaquera** [reply]
Mande el comando de  `mvn dependency:resolve ` e iba a sacar captura de pantalla cuando me di cuenta que ese comando resolvio las dependencias y ahora si nos las tomo correctamente.
Si no es mucha molestia, podrian ver este issue que pusimos previamente pero que todavia no obtuvimos respuesta.
https://github.com/dds-jv/foro/discussions/236
Desde ya muchas gracias!! :)

**Reply by nanderson87** [reply]
Claro, es que ese comando `mvn dependency:resolve` justamente trata de resolver las dependencias del proyecto y actualizar las mismas. Es probable que la hayan agregado en el IDE y por algún motivo solo faltaba actualizarlas.

Respecto a la discusión https://github.com/dds-jv/foro/discussions/236. Dale, les deje un comentario.

**Reply by jfloresmaquera** [reply]
Muchas gracias por responder!
